Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -1002,6 +959,7 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
     case options
     case experimentalFeatures
     case schemaDownloadConfiguration
+    case operationManifestConfiguration
```

Comment:
I'm open to doing that, I named it `operationManifestConfiguration` to keep with the existing naming format from `schemaDownloadConfiguration` and since the struct is called `OperationManifestConfiguration`